lib/matplotlib/figure.py

```diff
@@ -273,6 +273,13 @@ class Figure(Artist):
     def __str__(self):
         return "Figure(%gx%g)" % tuple(self.bbox.size)
 
+    def __repr__(self):
+        return "<{clsname} size {h:g}x{w:g} with {naxes} axes>".format(
```

Comment:
Capital Axes, though, referring to the class.

Reply:
done